scripts/generate-sync-bridge-client.py

```diff
@@ -0,0 +1,145 @@
+
+        # Create the import statement
+        run_coroutine_sync_import = cst.SimpleStatementLine(
+            body=[
+                cst.ImportFrom(
+                    module=cst.Attribute(
+                        value=cst.Attribute(
+                            value=cst.Name("derive_client"),
+                            attr=cst.Name("utils"),
+                        ),
+                        attr=cst.Name("asyncio_sync"),
+                    ),
+                    names=[cst.ImportAlias(name=cst.Name("run_coroutine_sync"))],
+                )
+            ],
+        )
+
+        # Find the position to insert (after the last import)
+        insert_pos = 0
+        for i, statement in enumerate(updated_node.body):
+            if isinstance(statement, (cst.SimpleStatementLine, cst.Import, cst.ImportFrom)):
+                insert_pos = i + 1
+            elif isinstance(statement, cst.EmptyLine):
+                continue
+            else:
+                break
+
+        new_body = list(updated_node.body)
+        new_body.insert(insert_pos, run_coroutine_sync_import)
+
+        return updated_node.with_changes(body=new_body)
+
+    def leave_ClassDef(
+        self,
+        original_node: cst.ClassDef,
+        updated_node: cst.ClassDef,
+    ) -> cst.ClassDef:
+        """Rename the class."""
+        if updated_node.name.value == "AsyncBridgeClient":
+            return updated_node.with_changes(name=cst.Name("BridgeClient"))
+        return updated_node
+
+    def _accesses_derive_bridge_light_account(self, node: cst.BaseExpression) -> bool:
+        """Check if the expression accesses derive_bridge.light_account."""
+        # Match pattern: derive_bridge.light_account.*
+        matcher = matchers.Attribute(
+            value=matchers.Attribute(
+                value=matchers.Name("derive_bridge"),
+                attr=matchers.Name("light_account"),
+            ),
+        )
+
+        # Check if the node itself matches
+        if matchers.matches(node, matcher):
+            return True
+
+        # If it's a Call, check the func
+        if isinstance(node, cst.Call) and matchers.matches(node.func, matcher):
+            return True
+
+        # Recursively check if any part of the expression contains the pattern
+        if isinstance(node, cst.Call):
+            return self._accesses_derive_bridge_light_account(node.func)
+        elif isinstance(node, cst.Attribute):
+            return self._accesses_derive_bridge_light_account(node.value)
+
+        return False
+
+
+def generate_sync_bridge_client():
+    async_bridge_client = PACKAGE_DIR / "_bridge" / "async_client.py"
+    sync_bridge_client = async_bridge_client.parent / "client.py"
+
+    source = async_bridge_client.read_text()
+    tree = cst.parse_module(source)
+    transformed = tree.visit(SyncConverter())
+    sync_bridge_client.write_text(transformed.code)
+    print(f"  ✓ Generated {sync_bridge_client}")
```

Comment:
inconsistent  ticks

Reply:
yes. There's small grey ones for each of the individual files, then one final green one at the end! :tada: